code/modules/mob/living/carbon/xenomorph/castes/king/king.dm

```diff
@@ -26,15 +26,15 @@
 /mob/living/carbon/xenomorph/king/generate_name()
 	switch(upgrade)
 		if(XENO_UPGRADE_ZERO)
-			name = "[hive.prefix]King ([nicknumber])"	 //Young
+			name = "[hive ? hive.prefix : "Disowned "]King ([nicknumber])"	 //Young
```

Comment:
i think it'd be awesome if you changed 'Disowned' to 'Feral' to symbolize how they're wild and hiveless

optional: darken hiveless xenos to differentiate them as well